aten/src/ATen/mps/MPSDevice.mm

```diff
@@ -99,8 +99,9 @@ static inline MTLLanguageVersion getMetalLanguageVersion(const id<MTLDevice>& de
     sampleGridWithSourceTensor:coordinateTensor:layout:normalizeCoordinates:relativeCoordinates:alignCorners:paddingMode:samplingMode:constantValue:name:)] == YES;
   static bool _macos_13_2_plus = [mpsCD instancesRespondToSelector:@selector(convolution3DWithSourceTensor:weightsTensor:descriptor:name:)] == YES;
   static bool _macos_13_3_plus = NO;
-  if (@available(macOS 13.3, *))
+  if (@available(macOS 13.3, *)) {
     _macos_13_3_plus = YES;
+  }
```

Comment:
As I've mentioned previously, this will not work, and I though the plan was to fix it with selector

Reply:
@malfet , this is an issue specifically for x86 side, it seems to work for Apple Silicon. Regardless we added this: https://github.com/pytorch/pytorch/pull/95866
The problem is sometimes in SU we don't have solid selectors to use. I think we need to start relying on ```@available``` to select the SUs